site/app/models/Config.php

```diff
@@ -95,7 +95,7 @@ class Config extends AbstractModel {
     /** @property @var string */
     protected $authentication;
     /** @property @var string */
-    protected $timezone = "America/New_York";
+    protected $timezone = 'America/New_York'; //Default to EST
```

Comment:
Remove comment as it's not really telling anything you can't just glean from looking at the default value. It's also wrong half the year when 'America/New_York' stands for EDT timezone (like right now).